metadata/config.yaml

```
@@ -37,6 +37,10 @@ CAPELLA_FORK_EPOCH: 256
 DENEB_FORK_VERSION: 0x05017000
 DENEB_FORK_EPOCH: 29696
 
+# Electra
+ELECTRA_FORK_VERSION: 0x06017000
+ELECTRA_FORK_EPOCH: 113152
```

```suggestion
ELECTRA_FORK_EPOCH: 115968
```

ACDC accepted option 2: https://github.com/ethereum/pm/issues/1265#issuecomment-2637778317